pkg/setting/setting.go

```diff
@@ -806,6 +809,10 @@ func (cfg *Cfg) Load(args *CommandLineArgs) error {
 		return err
 	}
 
+	if err := readDataSourcesSettings(iniFile, cfg); err != nil {
+		return err
+	}
```

Comment:
As the pattern otherwise is to use utility methods on Cfg instead of free functions, I think it's best to be consistent here and repeat the pattern:
```suggestion
	cfg.readDataSourceSettings()
```